rocAL/include/parameters/parameter_factory.h

```diff
@@ -104,4 +108,6 @@ class ParameterFactory {
     static ParameterFactory* _instance;
     static std::mutex _mutex;
     ParameterFactory();
+    std::vector<int64_t> _seed_vector;
+    int _seed_sequence_idx = 0;
```

Comment:
Why are seed changes present in this PR ?

Reply:
Please remove if it was unintentional

Reply:
I added it since it was related to parameterVX changes and the changes overlap

Reply:
This might be confusing I feel, since this is seed related and if it is not related to the ParameterVX change better to add in other

Reply:
Are we using _seed_vector and _seed_sequence_idx in this PR?